src/test/java/com/hotels/bdp/circustrain/bigquery/conversion/BigQueryToHiveTableConverterTest.java

```diff
@@ -51,50 +50,10 @@ public void withLocation() {
   }
 
   @Test
-  public void withSchema() {
-    Field integerField = Field.of("integer", LegacySQLTypeName.INTEGER);
-    Field stringField = Field.of("string", LegacySQLTypeName.STRING);
-    Field booleanField = Field.of("boolean", LegacySQLTypeName.BOOLEAN);
-    Field floatField = Field.of("float", LegacySQLTypeName.FLOAT);
-    Field dateField = Field.of("date", LegacySQLTypeName.DATE);
-    Field dateTimeField = Field.of("datetime", LegacySQLTypeName.DATETIME);
-    Field bytesField = Field.of("bytes", LegacySQLTypeName.BYTES);
-    Field timeStampField = Field.of("timestamp", LegacySQLTypeName.TIMESTAMP);
-    Field timeField = Field.of("time", LegacySQLTypeName.TIME);
-    Field numericField = Field.of("numeric", LegacySQLTypeName.valueOf("NUMERIC"));
-
-    Schema schema = Schema
-        .of(integerField, stringField, booleanField, floatField, dateField, dateTimeField, bytesField, timeStampField,
-            timeField, numericField);
+  public void withSchema() throws IOException {
+    String schema = new String(SchemaExtractorTest.getContentFromFileName("usa_names_schema.avsc"));
```

Comment:
A unit test class should just contain tests and if you're using it like this from another class it means it's doing too much. So my suggestion would be to create a small SchemaUtils class in the test package and move the `getContentFromFileName` method there.